lib/beaker-puppet/install_utils/puppet_utils.rb

```diff
@@ -21,11 +21,11 @@ module PuppetUtils
         #    'foss'
         def normalize_type type
           case type
-          when /(\A|-)(git)|(foss)(\Z|-)/
+          when /(\A|-)foss(\Z|-)/
             'foss'
           when /(\A|-)pe(\Z|-)/
             'pe'
-          when /(\A|-)aio(\Z|-)/
+          when /(\A|-)(git)|(aio)(\Z|-)/
```

Comment:
I think we can avoid doing this and conflating git/aio by calling these methods directly:

```ruby
  add_aio_defaults_on(hosts)
  add_puppet_paths_on(hosts)
```